README.rst

```diff
@@ -203,7 +203,7 @@ To use Azure IoT Central, you will need to create an Azure IoT Central app, crea
 
     from adafruit_azureiot import IoTCentralDevice
 
-    device = IoTCentralDevice(wifi, secrets["id_scope"], secrets["device_id"], secrets["key"])
+    device = IoTCentralDevice(wifi, secrets["id_scope"], secrets["device_id"], secrets["device_device_sas_key"])
```

Comment:
Agree with this change, pls. tag me when this is read for final review as the guide's documentation will need to be modified as well.

Reply:
> > I removed the neopixel stuff from simple examples. I find these "extras" confusing when learning a new library.
> 
> I'd prefer to keep this in, modify the behavior of the examples as little as possible.

I should clarify. I shouldn't have said I'm "removing" anything as I'm not changing the ESP32SPI samples. They use the wifi manager which takes a status light parameter.

`wifi = adafruit_esp32spi_wifimanager.ESPSPI_WiFiManager(esp, secrets, status_light)`

For boards with native networking, there is not a similar status_light to be passed.

`wifi.radio.connect(secrets["ssid"], secrets["password"])`

Reply:
Also the readme is a typo it will be `device_sas_key` not `device_device_sas_key`. 

I'll definitely do a pass through the docs as well.